tensorflow/python/ops/image_ops_impl.py

```diff
@@ -40,6 +40,7 @@
 from tensorflow.python.ops import variables
 from tensorflow.python.util import deprecation
 from tensorflow.python.util.tf_export import tf_export
+from tensorflow.python.client import session
```

Comment:
Implementations of TensorFlow operations are not allowed to depend on sessions.

Operations need to support eager execution, function building, and a few other contexts in which a session does not exist.

Also the goal of operation code is to build a graph that will be executed later; using a session here violates this and can have weird unforeseen consequences.